optimism.tokenlist.json

```diff
@@ -1002,6 +1002,28 @@
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
     },
+    {
+      "chainId": 1,
+      "address": "0xca1207647Ff814039530D7d35df0e1Dd2e91Fa84",
+      "name": "dHEDGE",
```

Comment:
Can you set the name as `dHedge DAO Token` both here and in the next entry for chainId 10 to match the deployed tokens on both networks.